layers/+emacs/helpful/packages.el

```diff
@@ -0,0 +1,28 @@
+;;; packages.el --- helpful layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Johnson Denen <johnson@johnsons-macbook-pro.local>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(defconst helpful-packages
+  '(helpful))
+
+(defun helpful/init-helpful ()
+  (use-package helpful
+    :defer t
```

Comment:
`:defer t` can be removed now, [all are deferred by default](https://github.com/syl20bnr/spacemacs/commit/29c78ce841ee5a99bb4077a2862afc96c8136107).

Reply:
For the record, we have reverted `:defer t` to be the default has it introduced lots of regression in the end and increased maintenance burden.